src/topology/algebra/infinite_sum.lean

```diff
@@ -493,6 +493,20 @@ by { simp [sub_eq_add_neg], exact hf.add hg.neg }
 lemma summable.sub (hf : summable f) (hg : summable g) : summable (λb, f b - g b) :=
 (hf.has_sum.sub hg.has_sum).summable
 
+lemma has_sum.update (hf : has_sum f a₁) (b : β) [decidable_eq β] (a : α) :
+  has_sum (update f b a) (a - f b + a₁) :=
+begin
+  convert ((has_sum_ite_eq b _).add hf),
```

Comment:
Probably `function.update_apply` can help with golfing here.

Reply:
I get stuck on a goal `ite (b' = b) a (f b') = ite (b' = b) (a - f b + f b') (f b')`, which isn't really any easier to resolve than the way I currently tackle the original goal.